### ISSUE TITLE
Iterate daily forecast

### ISSUE THREAD

**Issue by greg-does-weather** [weather-gov/weather.gov#680]
## Daily
[Annotated Mobile](https://www.figma.com/file/0dSABt9zCqoknaKua6DHmP/NOAA---Weather.gov?type=design&node-id=2196%3A31610&mode=design&t=bOEgGOPKzi4hAEyK-1)

[Annotated Desktop
](https://www.figma.com/file/0dSABt9zCqoknaKua6DHmP/NOAA---Weather.gov?type=design&node-id=2196%3A31710&mode=design&t=bOEgGOPKzi4hAEyK-1)

[Reference](https://www.figma.com/file/0dSABt9zCqoknaKua6DHmP/NOAA---Weather.gov?type=design&node-id=2196%3A33686&mode=dev&t=bOEgGOPKzi4hAEyK-1)

Component of #647

**Comment by greg-does-weather** [comment]
I know #704 is part of this. Should we create a separate ticket for the work to expand the hourly data in the daily forecast, or hold this one open?

**Comment by eric-gade** [comment]
@greg-does-weather I think we are waiting on design to flesh out what they want from the detailed forecast. @partly-igor might have better insight

**Comment by partly-igor** [comment]
@greg-does-weather and @eric-gade that's right, I'm going to be working on detailed weather this week, and it's a good bet that a number of those details will live under the "see more details" action. I think you could start on the interaction piece, but will leave it up to you how to best break it up and when to pick it up.

**Comment by eric-gade** [comment]
Closed by #704
